examples/AllTests/EventDispatcherTest.cpp

```diff
@@ -106,16 +98,8 @@ TEST(EventDispatcher, DifferentEventWithRegistrationDoesNotResultIntoCallback)
 
 TEST(EventDispatcher, RegisterTwoObserversResultIntoTwoCallsAndARegistrationNotification)
 {
-    mock()
-        .expectOneCall("notify")
-        .onObject(&observer)
-        .withParameterOfType("Event", "event", &event)
-        .withParameter("timeOutInSeconds", 10);
-    mock()
-        .expectOneCall("notify")
-        .onObject(&observer2)
-        .withParameterOfType("Event", "event", &event)
-        .withParameter("timeOutInSeconds", 10);
+    mock().expectOneCall("notify").onObject(&observer).withParameterOfType("Event", "event", &event).withParameter("timeOutInSeconds", 10);
```

Comment:
Is there no way to keep these formatted as they were?

Reply:
This expression is less that 180 columns, so the formatter puts it on a single line.

Reply:
This isn't so much about the length of the line but about the formatting of method chaining. I checked online also though and it seems to be a missing feature in the clang formatter at the moment. Which sucks. I'll need to think about this a bit... but my initial thought is probably that this is big enough to anyways turn the max line off completely and leave it to the choice of the author.